### ISSUE TITLE
GenServer connection process timeouts.

### ISSUE THREAD

**Issue by Caciquez** [betrybe/neo4_ecto#9]
Currently on `CI` pipelines enviroment the connection pool processes exits during the test.

```shell
Error:      test/neo4_ecto_test.exs:30
     ** (exit) exited in: GenServer.call(Bolt.Sips.Router, {:get_connection, :direct, :default}, 5000)
         ** (EXIT) no process: the process is not alive or there's no process currently associated with the given name, possibly because its application isn't started
     stacktrace:
       (elixir) lib/gen_server.ex:979: GenServer.call/3
       (bolt_sips) lib/bolt_sips.ex:118: Bolt.Sips.conn/2
       test/neo4_ecto_test.exs:16: Neo4EctoTest.__ex_unit_setup_0/1
       test/neo4_ecto_test.exs:1: Neo4EctoTest.__ex_unit__/2
```

**Comment by WLSF** [comment]
We're currently having these issues because we're using a CI Matrix and running 4 concurrent/parallel tests at the same time, for some reason the Bolt.Sips connection is getting killed at any point and it breaks the pipe.

**Comment by Caciquez** [comment]
Closing since it has been fixed on #19
